app/src/main/res/layout/walkthrough_topic_summary_view.xml

```diff
@@ -22,8 +22,8 @@
 
     <org.oppia.android.app.customview.LessonThumbnailImageView
       android:id="@+id/walkthrough_topic_thumbnail_image_view"
-      android:layout_width="match_parent"
-      android:layout_height="92dp"
+      android:layout_width="0dp"
+      android:layout_height="0dp"
```

Comment:
Why this change?

Reply:
I noticed that "First Test Topic"'s image looked different, and these were layout dimensions in another view I used to compare, I think it was "topic_summary_view.xml"

Reply:
Along with this change please use `android:lines="1"` and `android:ellipsize="end"` in `walkthrough_lesson_count_text_view` textview then these solutions will work correctly together.

Reply:
Done.